### DISCUSSION TITLE
I want to transfer you the repository

### DISCUSSION THREAD

**Discussion by starwalkn** [gotenberg/gotenberg#1034] [Q&A] [answered]
Hi! I have a [forked TCM client](https://github.com/dcaraxes/gotenberg-go-client) which I developed for a short time, and achieved that the client is now fully compatible with Gotenberg 8.x and has completely updated the documentation. Unfortunately, I don't have that much time right now to continue working on the client, so I would like to transfer the repository to https://github.com/gotenberg, where it can find great community support and will be clearly visible to Gotenberg users in the list of your repositories.

Unless, if you agree to accept the repository, you will have to change the username to Awesome Gotenberg)

**Comment by gulien** [comment, marked as answer]
Hello @dcaraxes,

Thank for you very much for your time working on the library 👍 

I also don't have the time to manage another library, so, for now, the best option is to seek for new maintainers directly in your repository 😄
